internal/backend/db/dbgorm/sessions_test.go

```diff
@@ -25,7 +25,7 @@ func (f *dbFixture) addSessionLogRecordAndAssert(t *testing.T, logr scheme.Sessi
 	findAndAssertCount[scheme.SessionLogRecord](t, f, expected, "session_id = ?", logr.SessionID)
 }
 
-func (f *dbFixture) listSessionsAndAssert(t *testing.T, expected int) []scheme.Session {
+func (f *dbFixture) listSessionsAndAssert(t *testing.T, expected int64) []scheme.Session {
```

Comment:
please add more tests
- with count only
- list session when there is no sessions + count only for the same case
- count in case of pagination (could u define page size = 1? and create 2 sessions?)